Paper.tex

```diff
@@ -1309,8 +1309,8 @@ \section{Recursive Length Prefix}\label{app:rlp}\hypertarget{rlp}{}
 We define the set of possible structures $\mathbb{T}$:
 \begin{eqnarray}
 \mathbb{T} & \equiv & \mathbb{L} \cup \mathbb{B} \\
-\mathbb{L} & \equiv & \{ \mathbf{t}: \mathbf{t} = ( \mathbf{t}[0], \mathbf{t}[1], ... ) \; \wedge \; \forall_{n < \lVert \mathbf{t} \rVert} \; \mathbf{t}[n] \in \mathbb{T} \} \\
-\mathbb{B} & \equiv & \{ \mathbf{b}: \mathbf{b} = ( \mathbf{b}[0], \mathbf{b}[1], ... ) \; \wedge \; \forall_{n < \lVert \mathbf{b} \rVert} \; \mathbf{b}[n] \in \mathbb{O} \}
+\mathbb{L} & \equiv & \{ \mathbf{t}: \mathbf{t} = ( \mathbf{t}[0], \mathbf{t}[1], ... ) \; \wedge \; \forall_{n < \lVert \mathbf{t} \rVert} \; \mathbf{t}[n] \in \mathbb{T} \; \wedge \; \lVert \mathbf{t} \rVert < 2^{256} \} \\
+\mathbb{B} & \equiv & \{ \mathbf{b}: \mathbf{b} = ( \mathbf{b}[0], \mathbf{b}[1], ... ) \; \wedge \; \forall_{n < \lVert \mathbf{b} \rVert} \; \mathbf{b}[n] \in \mathbb{O} \; \wedge \; \lVert \mathbf{b} \rVert < 2^{256} \}
```

Comment:
In both cases, it seems that in [Appendix B](https://ethereum.github.io/yellowpaper/paper.pdf#appendix.B), with B, AIUI, the input can be any arbitrary data array, not just one less than than the stack limit of 2^256. Looking at the definitions of R_b and R_l, this seems to match up.

Reply:
@jamesray1 so, do you think no limit is necessary?

Reply:
AIUI, no, it seems unnecessary, but you are welcome to convince me otherwise

Reply:
Right, we can define a function that's not injective.  We can later say it's injective for certain short inputs.